src/Symfony/Component/Mime/Address.php

```diff
@@ -77,7 +77,12 @@ public function getEncodedAddress(): string
 
     public function toString(): string
     {
-        return ($n = $this->getName()) ? $n.' <'.$this->getEncodedAddress().'>' : $this->getEncodedAddress();
+        return ($n = $this->getEncodedName()) ? $n.' <'.$this->getEncodedAddress().'>' : $this->getEncodedAddress();
+    }
+
+    public function getEncodedName(): string
+    {
+        return sprintf('"%s"', str_replace('/"/u', '\"', $this->getName()));
```

Comment:
This was `preg_replace`, `str_replace` doesn't use modifiers 😊 

FYI. https://www.php.net/addcslashes as the current implementation doesn't protect against double escaping.

Reply:
Yes of course I think I was tired 😅

Reply:
Done